packages/expo-cli/src/commands/generate-module/configureModule.ts

```diff
@@ -0,0 +1,284 @@
+import fse from 'fs-extra';
+import path from 'path';
+import walkSync from 'klaw-sync';
+import { ModuleConfiguration } from './ModuleConfiguration';
+
+const asyncForEach = async <T>(array: T[], callback: (value: T, index: number, array: T[]) => Promise<void>) => {
+  for (let index = 0; index < array.length; index++) {
+    await callback(array[index], index, array);
+  }
+};
+
+async function removeFiles(directoryPath: string, filenames: string[]) {
+  await asyncForEach(filenames, async filename => await fse.remove(path.resolve(directoryPath, filename)));
+}
+
+/**
+ * Renames files names
+ * @param directoryPath - directory that holds files to be renamed
+ * @param extensions - array of extensions for files that would be renamed, must be provided with leading dot or empty for no extension, e.g. ['.html', '']
+ * @param renamings - array of filenames and their replacers
+ */
+const renameFilesWithExtensions = async (directoryPath: string, extensions: string[], renamings: { from: string; to: string; }[]) => {
+  await asyncForEach(
+    renamings,
+    async ({ from, to }) =>
+      await asyncForEach(extensions, async extension => {
```

Comment:
why not use `for of` instead? eg: https://stackoverflow.com/a/37576787/659988

Reply:
No particular reason, I just went with `for each` syntax, but I have no strong opinion here 😉

Reply:
haha yeah it doesn't really matter. personally i prefer `for of` with async/await because it looks more readable imo